gstools/normalizer/methods.py

```diff
@@ -0,0 +1,359 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think this package should be called `transformers.py` instead of `methods.py`!

Reply:
I don't want users to confuse it with the `transform` subpackage.